sklearn/naive_bayes.py

```diff
@@ -153,7 +153,7 @@ class labels known to the classifier
     epsilon_ : float
         absolute additive value to variances
 
-    sigma_ : ndarray of shape (n_classes, n_features)
+    var_ : ndarray of shape (n_classes, n_features)
```

Comment:
We need to keep the `sigma_` entry and add `.. deprecated` to inform that it will be removed in the future.